### PR TITLE
Server command

### PR DESCRIPTION
**Pull request by FusseligerDev**

### CODE REVIEW

File: node/src/main/java/de/obsidiancloud/node/command/ServerCommand.java [hunk 1]
```diff
@@ -0,0 +1,252 @@
...
+
+                                final String keyArg = args[4];
+
+                                if (argCount == 5) {
+                                    executor.sendMessage(
+                                            "server "
+                                                    + serverArg
+                                                    + " set env set "
+                                                    + keyArg
+                                                    + " <value>");
```
Comment (FabiPunktExe) [inline node/src/main/java/de/obsidiancloud/node/command/ServerCommand.java:195]: mach `"§cUsage: " + getCommandPrefix() + "server " + serverArg + " set env set " + keyArg + " <value>" `

File: node/src/main/java/de/obsidiancloud/node/command/ServerCommand.java [hunk 2]
```diff
@@ -0,0 +1,252 @@
...
+
+                                newEnvironmentVariables.put(keyArg, args[5]);
+
+                                server.setEnvironmentVariables(newEnvironmentVariables);
+                            }
+
+                            case "remove" -> {
+                                if (argCount == 4) {
+                                    executor.sendMessage(
+                                            "server " + serverArg + " set env remove <key>");
```
Comment (FabiPunktExe) [inline node/src/main/java/de/obsidiancloud/node/command/ServerCommand.java:211]: mach `"§cUsage: " + getCommandPrefix() + "server " + serverArg + " set env remove <key>" `

File: node/src/main/java/de/obsidiancloud/node/command/ServerCommand.java [hunk 3]
```diff
@@ -0,0 +1,252 @@
...
+                                        new HashMap<>(server.getData().environmentVariables());
+
+                                newEnvironmentVariables.remove(args[4]);
+
+                                server.setEnvironmentVariables(newEnvironmentVariables);
+                            }
+
+                            default ->
+                                    executor.sendMessage(
+                                            "server " + serverArg + " set env <set|remove>");
```
Comment (FabiPunktExe) [inline node/src/main/java/de/obsidiancloud/node/command/ServerCommand.java:237]: mach `"§cUsage: " + getCommandPrefix() + "server " + serverArg + " set env <set|remove>" `

File: node/src/main/java/de/obsidiancloud/node/command/ServerCommand.java [hunk 4]
```diff
@@ -0,0 +1,252 @@
...
+                        }
+                    }
+
+                    default ->
+                            executor.sendMessage(
+                                    "server " + serverArg + " set <name|autostart|memory|env>");
```
Comment (FabiPunktExe) [inline node/src/main/java/de/obsidiancloud/node/command/ServerCommand.java:243]: mach `"§cUsage: " + getCommandPrefix() + "server " + serverArg + " set <name|autostart|memory|env>"`

File: node/src/main/java/de/obsidiancloud/node/command/ServerCommand.java [hunk 5]
```diff
@@ -0,0 +1,252 @@
...
+                }
+            }
+
+            default ->
+                    executor.sendMessage(
+                            "server " + serverArg + " <start|stop|restart|kill|delete|set>");
```
Comment (FabiPunktExe) [inline node/src/main/java/de/obsidiancloud/node/command/ServerCommand.java:249]: mach `"§cUsage: " + getCommandPrefix() + "server " + serverArg + " <start|stop|restart|kill|delete|set>"`

**Review by FabiPunktExe — CHANGES_REQUESTED** [review]

**Review by FabiPunktExe — APPROVED** [review]
code ist gut dass ich orgasmus bekomme

**Comment by FusseligerDev** [comment]
👍

**Pull request merged**